index.html

```diff
@@ -222,7 +222,7 @@ <h2 class="view">I'm always<br>
             <hr>
             <input type="text" class="input" placeholder="Email address">
```

Comment:
- [ ] Following the project requirements, the email input's `type` should equal `email`